internal/configs/version2/http.go

```diff
@@ -71,6 +71,7 @@ type Server struct {
 	EgressMTLS                *EgressMTLS
 	PoliciesErrorReturn       *Return
 	Namespace                 string
+	Name                      string
```

Comment:
Would it make sense to rename these variables to `VSName` and `VSNamespace`? That will be consistent with `VSR*` variables. Also, the `Server.Name` field is a bit confusing because we have the `ServerName` variable, so `Server.VSName` will clear that confusion.